omnibus/package-scripts/agent/postinst

```diff
@@ -91,23 +91,37 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
         fi
 
         # Setup SELinux policy and label if SELinux detected on the host
+        # FIXME: Refactor warning messages
         if command -v semodule >/dev/null 2>&1 && [ -f "$INSTALL_DIR/embedded/bin/system-probe" ]; then
-            echo "Loading SELinux policy module for datadog-agent."
+            echo "Loading SELinux policy module for system-probe."
             semodule -v -i $CONFIG_DIR/selinux/system_probe_policy.pp >/dev/null 2>&1
             if [ "$?" != "0" ]; then
-                echo "The SELinux policy module for datadog-agent is not compatible with this host."
-                echo "Please install or update the selinux-policy-default package and then reinstall the datadog-agent package to configure SELinux for the Datadog Agent."
-                echo "Skipping SELinux configuration."
+                echo "Couldn’t load system-probe policy."
+                echo "To be able to run system-probe on your host, please install or update the"
+                echo "selinux-policy-default and policycoreutils-python-utils packages."
+                echo "Then run the following commands:"
+                echo "    semodule -i $CONFIG_DIR/selinux/system_probe_policy.pp"
+                echo "    semanage fcontext -a -t system_probe_t $INSTALL_DIR/embedded/bin/system-probe"
+                echo "    restorecon -v $INSTALL_DIR/embedded/bin/system-probe"
             else
                 echo "Labeling SELinux type for the system-probe binary."
                 if command -v semanage >/dev/null 2>&1 && command -v restorecon >/dev/null 2>&1;then
                     semanage fcontext -a -t system_probe_t $INSTALL_DIR/embedded/bin/system-probe && restorecon -v $INSTALL_DIR/embedded/bin/system-probe
                     if [ "$?" != "0" ]; then
-                        echo "Could not relabel the system-probe binary with type system_probe_t."
```

Comment:
We didn't have instructions here, now we do. Why the change? I guess it might be worth creating a function that prints these, since we have essentially the same instructions three times.

Reply:
Ok, we can do this for 7.19 then.